### PR TITLE
Add new metrics to explore (with placeholder content)

### PR DESCRIPTION
**Pull request by chasulin**
Trello: https://trello.com/c/BxIGkm4g/425-trends-include-community-level-sub-metrics-as-drop-down-options

### CODE REVIEW

**Comment by pnavarrc** [comment]
[Trends: Include Community Level - Sub-Metrics as Drop Down Options](https://trello.com/c/BxIGkm4g/425-trends-include-community-level-sub-metrics-as-drop-down-options)

File: src/components/Explore/utils.ts [hunk 1]
```diff
@@ -366,6 +385,53 @@ export const exploreMetricData: {
       },
     ],
   },
+
+  // Todo (update with real content)
```
Comment (pnavarrc) [inline src/components/Explore/utils.ts:389]: Maybe we can start using `TODO(8.2)` to make them easier to find later? I can comb the feature branch to find 8.2 TODOs and update the comments

**Review by pnavarrc — APPROVED** [review]
Looks good!

**Review by faihegberg — APPROVED** [review]

File: src/components/Explore/utils.ts (hunk 1 above)
Comment (chasulin) [reply src/components/Explore/utils.ts:389]: good idea

**Pull request merged**